Load appliances from settings.txt

theis999 opened the pull request:
- appliance_upsert_function now also appends the appliance to the settings file.
- appliance_remove_function now also removes the appliance from the settings file. 
- Main loads appliances from the settings file before starting the menu.

_____

This changes ensures that appliances are saved between runs. It does so by saving changes done to the appliances in the settings.txt file, and loading the appliances from that file too.

Since settings follows the format <key value>, which only allows for unique keys and only allows one value per key: each appliance will use 3 lines in the settings file. One for it's name, one for wh (watt hours) and one for runtime (runTime).

MortenSOlesen commented on PowerHouse/Main.c:
```diff
@@ -34,6 +36,39 @@ void run_menu (int startup_menu) {
+		(!memcmp(str + str_len - suffix_len, suffix, suffix_len));
+}
+
+void insert_applianaces_from_settings()
```
Mindre stavefejl: `applianaces` vs `appliances`

MortenSOlesen commented on PowerHouse/Main.c:
```diff
@@ -34,6 +36,39 @@ void run_menu (int startup_menu) {
+void insert_applianaces_from_settings()
+{
+	setting* settings = GetAllSettings();
+	for (int i = 0; i < 50; i++)
```
hvorfor 50 gange?

theis999 committed:
appliance_upsert_function now also appends the appliance to the settings file.
appliance_remove_function now also removes the appliance from the settings file.
Main loads appliances from the settings file before starting the menu.

theis999 committed:
Added GetSettingsCount() and used it to iterate over settings to appliance in main.
